### PR TITLE
Json file support

### PR DESCRIPTION
**Pull request by JohnPTobe**
Adding method to read json file and return a string

### CODE REVIEW

File: util/file.go [hunk 1]
```diff
@@ -165,3 +168,23 @@ func ReadLinesFromFile(filename string) ([]string, error) {
 
 	return lines, nil
 }
+
+func ReadJsonFile(filename string) (string, error) {
```
Comment (gisjedi) [inline util/file.go]: I don't see any evidence of this function being used. Is this PR complete or is it intended to just be used in consuming projects?

File: constants/constants.go [hunk 2]
```diff
@@ -135,6 +135,12 @@ const RepeatFlag = "repetitions"
 //ShortRepeatFlag - shorthand flag for repetitions
 const ShortRepeatFlag = "rep"
 
+//VersionFlag defines version of seed spec to use
+const VersionFlag = "version"
```
Comment (gisjedi) [inline constants/constants.go]: I don't see any evidence of this const being used. Is it being used in consuming project?

File: util/file.go (hunk 1 above)
Comment (JohnPTobe) [reply util/file.go]: This is for json file input support in seed-cli

File: constants/constants.go (hunk 2 above)
Comment (JohnPTobe) [reply constants/constants.go]: This is for specifying the seed spec version in seed-cli commands

Comment (JohnPTobe) [reply constants/constants.go]: Good point, I initially thought there'd be shared constants but it's just been a big pain to have them in a separate project

**Comment by JohnPTobe** [comment]
I've removed the seed-cli constants and json method.

**Pull request merged**